scripts/update_metadata.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+'''
+@author: Patrick Cleeve
+
+This is a script to update the MD_POS_COR for the stage-bare component while odemis is running.
+
+run as:
+python3 ./scripts/update_metadata.py 
+
+and follow the instructions in the terminal.
+'''
+
+
+from odemis import model
+
+def main():
+
+    print(f"Calibration metadata updater")
+
+    stage = model.getComponent(role="stage-bare")
+    print(f"-"*20)
+    print(f"Current stage-bare component: {stage}")    
+    print(f"Current calibration metadata: {stage.getMetadata()[model.MD_POS_COR]}")
+
+    # get x , y values from user
+    x = input(f"Enter new x calibration value: ")
+    y = input(f"Enter new y calibration value: ")
+    print("-"*20)
+
+    # check they are floats
+    try:
+        x = float(x)
+        y = float(y)
+    except  ValueError:
+        print("Invalid input. Please enter a float. Calibration data has not been updated.")
+        return
+    
+
+    # get user confirmation for values
+    print(f"Change to new calibration metadata: {[x, y]}")
+
+    ret = input(f"Confirm? (y/n): ")
```

Comment:
The default is "n", so the convention is to indicate it by using an uppercase. -> "y/N".